src/main/java/org/sefglobal/scholarx/model/BaseScholarxModel.java

```diff
@@ -16,7 +16,7 @@
 
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
-    private int id;
+    private long id;
```

Comment:
It was a int variable on the class diagram. Which one is better? Int or long

Reply:
let's use long it's better for the long term. And there is no performance difference